webapp/config/routes/annotations.yaml

```diff
@@ -1,3 +1,4 @@
 controllers:
-    resource: ../../src/Controller/
+    resource: ../../src/Controller/**/*
```

Comment:
Note: this change is needed because otherwise the API would also be exposed on `/`